pkg/tsdb/elasticsearch/response_parser_frontend_test.go

```diff
@@ -812,24 +812,23 @@ func TestHistogramSimple(t *testing.T) {
 
 	require.Len(t, result.response.Responses, 1)
 	frames := result.response.Responses["A"].Frames
-	// require.Len(t, frames, 3) // FIXME
+	require.Len(t, frames, 1)
```

Comment:
On frontend we have `frame.length` field that tracks how many values are in fields. On backend we don't have this, so we need to check for `require.Equal(t, fields[0].Len(), 3)` which we are doing below. `require.Len(t, frames, 1)` is just another check.